killrweather-dashboard/app/com/datastax/killrweather/DashboardApiActor.scala

```diff
@@ -0,0 +1,47 @@
+package com.datastax.killrweather
+
+import akka.actor.{Actor, ActorLogging}
+import akka.cluster.Cluster
+import akka.util.Timeout
+import com.datastax.killrweather.DashboardApiActor.GetWeatherStationWithPrecipitation
+import com.datastax.killrweather.Weather.{DailyPrecipitation, WeatherStation}
+import com.datastax.killrweather.WeatherEvent.{GetWeatherStations, GetTopKPrecipitation, GetWeatherStation}
+import akka.pattern.{ ask, pipe}
+import com.datastax.killrweather.service.{DayPrecipitation, WeatherStationInfo}
+import play.Logger
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+
+class DashboardApiActor extends Actor with ActorLogging {
+
+  //todo: make this part of the request?
+  val DefaultWeatherEvents = 50
```

Comment:
Make configurable with 50 as the default. This belongs in a Settings kind of class that picks up config.